lib/std/build.zig

```diff
@@ -1212,6 +1213,8 @@ pub const LibExeObjStep = struct {
     is_linking_libc: bool = false,
     vcpkg_bin_path: ?[]const u8 = null,
 
+    /// This may be set in order to override the default install directory
+    override_dest_dir: ?InstallDir,
```

Comment:
Doesn't really seem like a function to set this variable is necessary, users of the api can just set the field directly.